Client/Application/AppDelegate.swift

```diff
@@ -294,6 +294,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             self.profile?.shutdown()
             application.endBackgroundTask(taskId)
         }
+
+        // Workaround for crashing in the background when <select> popovers are visible.
+        // See http://openradar.appspot.com/radar?id=5033179731722240
+        let jsBlurSelect = "if (document.activeElement.tagName === 'SELECT') { document.activeElement.blur(); }"
+        tabManager.selectedTab?.webView?.evaluateJavaScript(jsBlurSelect, completionHandler: nil)
```

Comment:
In the vein of lifecycle bugs: I think you might want to capture `selectedTab` or `webView` itself outside of this async block. It doesn't feel particularly right to be firing off some background work in response to a UI thread change — a background callback — that just runs against a potentially different UI state.